alica_engine/src/engine/RunnableObject.cpp

```diff
@@ -50,7 +52,49 @@ void RunnableObject::start(RunningPlan* rp)
     }
     ++_signalState;
     _signalContext.store(rp);
-    _engine->editScheduler().schedule(std::bind(&RunnableObject::doInit, this));
+    if(_requiresParameters) {
```

Comment:
does plan designer or backend enforce that this flag can't be set for master plan?

Reply:
yes